src/resolvers/viewsInfo.ts

```diff
@@ -21,9 +20,10 @@ class BatchedVideoViewsArgs {
 
 @ArgsType()
 class MostViewedVideosArgs {
```

Comment:
We have a couple of classes specified with exact same field - `period` and `limit`. We could just use one class for all of those right?